src/transformers/models/imagegpt/configuration_imagegpt.py

```diff
@@ -14,7 +14,14 @@
 # limitations under the License.
 """ OpenAI ImageGPT configuration"""
 
+from collections import OrderedDict
+from typing import Any, Mapping, Optional
+
+from transformers.feature_extraction_utils import FeatureExtractionMixin
```

Comment:
Ok. One small doubt, Should I put `Any`, `Mapping` etc under `TYPE_CHECKING` clause?

Reply:
Also, after putting `TensorType` under `TYPE_CHECKING` clause, I am getting following error
```bash
RuntimeError: Failed to import transformers.models.imagegpt.configuration_imagegpt because of the following error (look up to see its traceback):
name 'TensorType' is not defined
```

Reply:
It's OK to leave `Any` etc as proper imports - the `TYPE_CHECKING` clause is usually reserved for imports coming from the `transformers` library :)

To fix your issue, you need to put `TensorType` in quotes, i.e. `"TensorType"`. You can find more information about "forward referencing" here: https://peps.python.org/pep-0484/#forward-references

Reply:
Thank you for replying and clearing the doubt.